worlds/sc2/Items.py

```diff
@@ -68,175 +213,807 @@ def get_full_item_list():
-    ItemNames.PLANETARY_FORTRESS: ItemData(612 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 12, SC2Race.TERRAN, classification=ItemClassification.progression),
-    ItemNames.PERDITION_TURRET: ItemData(613 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 13, SC2Race.TERRAN, classification=ItemClassification.progression),
-    ItemNames.PREDATOR: ItemData(614 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 14, SC2Race.TERRAN, classification=ItemClassification.filler),
-    ItemNames.HERCULES: ItemData(615 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 15, SC2Race.TERRAN, classification=ItemClassification.progression),
-    ItemNames.CELLULAR_REACTOR: ItemData(616 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 16, SC2Race.TERRAN),
-    ItemNames.PROGRESSIVE_REGENERATIVE_BIO_STEEL: ItemData(617 + SC2WOL_ITEM_ID_OFFSET, "Progressive Upgrade", 4, SC2Race.TERRAN, quantity=2),
-    ItemNames.HIVE_MIND_EMULATOR: ItemData(618 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 18, SC2Race.TERRAN, ItemClassification.progression),
-    ItemNames.PSI_DISRUPTER: ItemData(619 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 19, SC2Race.TERRAN, classification=ItemClassification.progression),
+    ItemNames.VALKYRIE_ENHANCED_CLUSTER_LAUNCHERS: ItemData(379 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 17,
+                                                            SC2Race.TERRAN, parent_item="Valkyrie", origin={"ext"},
+                                                            description="""
+                                                            Valkyrie fires 2 additional rockets each volley.
+                                                            """),
+    ItemNames.VALKYRIE_SHAPED_HULL: ItemData(380 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 20, SC2Race.TERRAN,
+                                             classification=ItemClassification.filler, parent_item="Valkyrie",
+                                             origin={"ext"},
+                                             description="""
+                                             Increases Valkyrie life by 50.
+                                             """),
+    ItemNames.VALKYRIE_BURST_LASERS: ItemData(381 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 21, SC2Race.TERRAN,
+                                              parent_item="Valkyrie", origin={"ext"},
+                                              description="""
+                                              Valkyries are equipped with Burst Lasers to attack ground units.
+                                              """),
+    ItemNames.VALKYRIE_AFTERBURNERS: ItemData(382 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 22, SC2Race.TERRAN,
+                                              classification=ItemClassification.filler, parent_item="Valkyrie",
+                                              origin={"ext"},
+                                              description="""
+                                              Allows Valkyries a quick burst of movement speed.
```

Comment:
Suggestion:
> Temporarily increases the Valkyrie's movement speed by 70%.

(based on GameStrings, which give a precise time. However, game speed being what it is, "temporary" seems better.)